src/main.ts

```diff
@@ -3,10 +3,19 @@ import { routes } from "./routes/index.ts";
 import { Innertube, UniversalCache } from "youtubei.js";
 import { poTokenGenerate } from "./lib/jobs/potoken.ts";
 import { USER_AGENT } from "bgutils";
-import { konfigLoader } from "./lib/helpers/konfigLoader.ts";
 import { retry } from "jsr:@std/async";
-import type { HonoVariables } from "./lib/types/HonoVariables.ts";
 import type { BG } from "bgutils";
+import type { HonoVariables } from "./lib/types/HonoVariables.ts";
+
+import { parseConfig } from "./lib/helpers/config.ts";
+const config = await parseConfig();
+
+if (
```

Comment:
wondering whenever we could have the validation in config.ts directly? could zod handle that?

Reply:
As discussed on matrix, going with the current implementation. Secret key is always required, verify_requests is optional. No change needed.